app/inbound/save-payment-request.js

```diff
@@ -0,0 +1,9 @@
+const { v4: uuidv4 } = require('uuid')
+const db = require('../data')
+
+const savePaymentRequest = async (paymentRequest, transaction) => {
+  delete paymentRequest.paymentRequestId
+  await db.paymentRequest.create({ ...paymentRequest, referenceId: uuidv4() }, { transaction })

```

Comment:
don't think we should override the `referenceId` as then we're just potentially making it harder to deal with duplicates?  Depending on the outcome of the duplicate check discussion.